### ISSUE TITLE
Some File Input examples are broken

### ISSUE THREAD

**Issue by jaredcunha** [uswds/uswds-site#1055]
Two of the File Input examples appear to be broken and are displaying the raw HTML
<img width="636" alt="Screen Shot 2020-09-18 at 2 21 21 PM" src="https://user-images.githubusercontent.com/1094951/93631918-6c093f00-f9ba-11ea-874c-d56886c194ce.png">

**Comment by thisisdano** [comment]
Yeah, saw this — on my immediate to to list!

**Comment by mejiaj** [comment]
Source in browser

```html
<div class="usa-form-group">
  <label class="usa-label" for="file-input-multiple">Input accepts multiple files</label>
  <span class="usa-hint" id="file-input-multiple-hint">Select one or more files</span>
  &lt;input id="file-input-multiple"
    class="usa-file-input"
    type="file"
    name="file-input-multiple"
    aria-describedby="file-input-multiple-hint"multiple/&gt;
</div>
```

**Comment by mejiaj** [comment]
This was fixed in uswds/uswds/pull/3905.
